docs/en/wallet-instance-attestation.rst

```diff
@@ -2,56 +2,632 @@
+
+Verifiable through the following public key (Wallet Provider Public Key)
+obtained within the Entity Configuration of the Wallet Provider,
+attested in the related trust chain.
+
+.. code-block:: javascript
+
+  {
+  "kty": "EC",
+  "x": "qrJrj3Af_B57sbOIRrcBM7br7wOc8ynj7lHFPTeffUk",
+  "y": "1H0cWDyGgvU8w-kPKU_xycOCUNT2o0bwslIQtnPU6iM",
+  "crv": "P-256"
+  }
+
+
+Format of the Wallet Provider Entity Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Wallet Provider Entity Configuration is a JWT containing all the metadata
+relating to the Wallet Provider such as the public keys, the supported algorithms,
+the type of entity (in addition to the Wallet Provider, for example, it could also
```

Comment:
Probably this would be moved to the wallet solution section, we'll discuss on that later on in the revision phase